core/contracts/oracle/implementation/Voting.sol

```diff
@@ -339,10 +339,25 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface, Encrypte
         emit VoteCommitted(msg.sender, currentRoundId, identifier, time);
     }
 
+    /**
+     * @notice Snapshot the current round's token balances and lock in the inflation rate and GAT.
```

Comment:
nit: we should probably note that this function will revert unless it's called during the reveal period.

Reply:
sure & added.